sender.go

```diff
@@ -268,6 +247,50 @@ func (s *Sender) trySend(ctx context.Context, evt eventer) error {
 	}
 }
 
+func (s *Sender) handleAmqpError(ctx context.Context, err error) error {
```

Comment:
Please add a comment here, explaining this method. Can be similar to what the python description has.